packages/cf-style-const/src/variables.js

```diff
@@ -4,6 +4,13 @@ const colorOffsetLight = 0.15;
 const colorOffsetDark = 0.07;
 
 export default {
+  breakpoints: {
+    mobile: '@media (min-width: 13.6em)',
+    mobileWide: '@media (min-width: 30.4em)',
```

Comment:
They come from our existing SASS and PostCSS variables files. I don't like it, but I also don't want to change it for fear of making the migration even harder.